core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -128,20 +106,9 @@ public void handleResolvedAddresses(
    * @param servers the resolved server addresses, never empty.
    * @param attributes extra metadata from naming system.
    */
-  @SuppressWarnings("deprecation")
   public void handleResolvedAddressGroups(
       List<EquivalentAddressGroup> servers, Attributes attributes) {
-    ArrayList<ResolvedServerInfoGroup> serverInfoGroups =
-        new ArrayList<ResolvedServerInfoGroup>(servers.size());
-    for (EquivalentAddressGroup eag : servers) {
-      ResolvedServerInfoGroup.Builder serverInfoGroupBuilder =
-          ResolvedServerInfoGroup.builder(eag.getAttributes());
-      for (SocketAddress addr : eag.getAddresses()) {
-        serverInfoGroupBuilder.add(new ResolvedServerInfo(addr));
-      }
-      serverInfoGroups.add(serverInfoGroupBuilder.build());
-    }
-    handleResolvedAddresses(serverInfoGroups, attributes);
+    throw new UnsupportedOperationException();
```

Comment:
This should be made abstract, since it must be implemented.

Reply:
Done.

Reply:
Not done yet?